Task 1/NameDayOfDate/NameDayOfDate.js

```diff
@@ -0,0 +1,15 @@
+
+// The days of the week are: "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"
+function getDayName(dateString) {
+    let dayName;
+    // Write your code here
+     const date = new Date(dateString);
+     
+     const options = {
+      weekday: 'long'
+    };
+  
+    dayName = new Intl.DateTimeFormat('en-Us', options).format(date);
+
+    return dayName;
+}
```

Comment:
It's a good practice to add a newline at the end of file!

Reply:
Sure